scripts/beaker_submit_gpu_tests.py

```diff
@@ -0,0 +1,102 @@
+"""
+Usage: python -m scripts.beaker_submit_gpu_tests NAME COMMIT_SHA
+"""
+
+import signal
+import sys
+
+import rich
+from rich import pretty, print, traceback
+
+rich.get_console().width = max(rich.get_console().width, 180)
+pretty.install()
+traceback.install()
+
+from beaker import (  # noqa: E402
+    EnvVar,
+    ExperimentSpec,
+    ImageSource,
+    ResultSpec,
+    TaskContext,
+    TaskResources,
+    TaskSpec,
+)
+
+from .beaker_common import (  # noqa: E402
+    BEAKER_CLOUD_CLUSTER,
+    BEAKER_IMAGE,
+    BEAKER_ON_PREM_CLUSTERS,
+    beaker,
+)
+
+
+class TermInterrupt(Exception):
+    pass
+
+
+def handle_sigterm(sig, frame):
+    raise TermInterrupt
```

Comment:
What's this for?

Reply:
GH Actions sends a SIGTERM when a job is cancelled, which Python doesn't catch by default.